languages/go/concepts/methods/about.md

```diff
@@ -1 +1,61 @@
-TODO: add information on methods concept
+# methods
+
```

Comment:
We'll automatically add this header on the website

```suggestion
```

Reply:
Cool